convert2rhel/backup.py

```diff
@@ -351,7 +352,21 @@ def restore(self):
 class NewRestorableFile(RestorableChange):
     def __init__(self, filepath):
         super(NewRestorableFile, self).__init__()
+
```

Comment:
Maybe we should normalize the path? I'm not entirely convinced we should bother, but if we're trying to catch mistakes during development, it would add some bulletproofing.

Reply:
I think it should be fine without it for now. The checking for an absolute path should be enough for now. Maybe if we get some issue from a user, we can investigate it more.